generators/server/templates/src/test/resources/META-INF/services/reactor.blockhound.integration.BlockHoundIntegration.ejs

```diff
@@ -0,0 +1 @@
+<%= packageName %>.config.JHipsterBlockHoundIntegration
```

Comment:
Maybe use [`@AutoService`](https://github.com/reactor/BlockHound/blob/master/docs/custom_integrations.md#custom-integrations) ?

Reply:
I'm not sure adding yet another dependency and another annotation processor is worth the value here, for something that is supposed to be a temporary workaround.

Reply:
Adding a file that will need to be cleaned up later doesn't seem better.

Reply:
Does really cleaning one file (`JHipsterBlockHoundIntegration.java`) or two files (`JHipsterBlockHoundIntegration.java` + meta-inf service declaration) makes a difference?

On the other hand, we have to deal with adding a new dependency (`com.google.auto.service:auto-service-annotations`), a new annotation processor (`com.google.auto.service:auto-service`) and a new property for the version that will have to be updated from time to time, for 2 build systems (gradle and maven).

I'm really not convinced here, in this context, with the use of `@Autoservice`, but I can add it others think it's the better solution.
@jhipster/developers what do you think?

Reply:
Tests pass - ship it! 🚢